### PR TITLE
Bump spades to 3.11.0

### PR DESCRIPTION
**Pull request by druvus**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/spades/meta.yaml [hunk 1]
```diff
@@ -21,7 +22,7 @@ requirements:
 
 test:
   commands:
-    - spades.py --test
+    - spades.py --version
```
Comment (notestaff) [inline recipes/spades/meta.yaml]: spades.py --test does not actually take long, so maybe worth keeping?

File: recipes/spades/meta.yaml [hunk 2]
```diff
@@ -21,7 +22,7 @@ requirements:
...
 
 about:
   home: http://bioinf.spbau.ru/en/spades
```
Comment (notestaff) [inline recipes/spades/meta.yaml]: spades homepage has moved to http://cab.spbu.ru/software/spades/

File: recipes/spades/meta.yaml [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
-  md5: fb58d48a0545211aab91f43b7d17ad1b # [linux]
-  fn: SPAdes-3.10.1-Darwin.tar.gz                                             # [osx]
-  url: http://spades.bioinf.spbau.ru/release3.10.1/SPAdes-3.10.1-Darwin.tar.gz # [osx]
-  md5: f23bbad7921cff2c283e6589ccee83a2 # [osx]
+  fn: SPAdes-{{ version }}-Linux.tar.gz                                              # [linux]
+  url: http://spades.bioinf.spbau.ru/release{{ version }}/SPAdes-{{ version }}-Linux.tar.gz  # [linux]
+  sha256: 5d362ea40bfc0fedfbdf7d17504f38f63e7e7e23b29fef89f89273313115db5f # [linux]
+  fn: SPAdes-{{ version }}-Darwin.tar.gz                                             # [osx]
+  url: http://spades.bioinf.spbau.ru/release{{ version }}/SPAdes-{{ version }}-Darwin.tar.gz # [osx]
+  sha256: 9e6accf2258c379436e92f37f920e4dc8fa4a2da9a9813215709a9723bf15d87 # [osx]
```
Comment (notestaff) [inline recipes/spades/meta.yaml]: Downloading binaries for spades has been working fine.  But in general, I remember being told that there's a preference for building things from source?  ( What are the exact reasons for preferring builds from source?  @bgruening @johanneskoester Maybe clarify this in https://bioconda.github.io/guidelines.html ? )

**Comment by notestaff** [comment]
@druvus maybe go back to downloading binaries for now?  let's get in a recipe that works, and later can see if can make compiling from source work.

**Comment by druvus** [comment]
@notestaff Agree. I am not able to do anuthing this week so could you please take over and submit the the simply bump based on binaries.

**Pull request closed without merging**